clients/graphql-kotlin-client-jackson/build.gradle.kts

```diff
@@ -7,6 +7,8 @@ plugins {
 dependencies {
     api(projects.graphqlKotlinClient)
     api(libs.jackson)
+    testImplementation("org.junit.jupiter:junit-jupiter")
```

Comment:
can we move this to libs toml file and then add it as a replacement for https://github.com/ExpediaGroup/graphql-kotlin/blob/master/buildSrc/src/main/kotlin/com.expediagroup.graphql.conventions.gradle.kts#L147-L148?

Reply:
Not really: https://github.com/junit-team/junit5-samples/blob/r5.10.2/junit5-jupiter-starter-gradle/build.gradle
But I was able to improve your dependencies and manage them better (by using the JUnit BOM) 🙂

Reply:
Yes, I was able to manage the junit-bom and declare all 3 dependencies in buildSrc.